src/ldpl.h

```diff
@@ -42,11 +46,15 @@ struct compiler_state{
     void add_var_code(string code){
         this->variable_code.push_back(code);
     }
-    void add_code(string code, unsigned int line_num = 0){
-	    auto& output = current_subprocedure == "" ? this->output_code : this->subroutine_code;
+    void add_code(string code) {
+        auto& output = current_subprocedure == "" ? this->output_code : this->subroutine_code;
+        output.push_back(code);
+    }
+    void add_code(string code, code_location where) {
```

Comment:
I think you can delete `where` and use `this->where`.

Reply:
I'll leave it for now. I was thinking about proper use of `#line` and there are some cases of calling `add_code(code)` without line number where we probably do not want to mess with `#line` once this feature is enabled. For now those two variants of `add_code` do the same thing but I hope we can enable it soon.